MediaWikiKit/MediaWikiKit/MWKSearchResult.m

```diff
@@ -20,10 +20,10 @@ + (NSValueTransformer*)wikidataDescriptionJSONTransformer {
 
 + (NSDictionary*)JSONKeyPathsByPropertyKey {
     return @{
-               @"displayTitle": @"title",
-               @"articleID": @"pageid",
-               @"thumbnailURL": @"thumbnail.source",
-               @"wikidataDescription": @"terms.description"
+               WMF_SAFE_KEYPATH(MWKSearchResult.new, displayTitle): @"title",
```

Comment:
When you made the suggestion of using `WMF_SAFE_KEYPATH` on my patch, I gave a reason about why I didn't think this was a good idea to use in in the context of Mantle methods:
https://github.com/wikimedia/wikipedia-ios/pull/71#discussion_r37452046